### PR TITLE
Refactoring and responsiveness

### PR DESCRIPTION
**Pull request by SinchanaMS**
1. Code cleanup
2. Responsiveness

### Preview:

https://user-images.githubusercontent.com/94738532/169292688-7b832c9f-6ead-4f67-ac79-1c3b2be337af.mp4

### CODE REVIEW

File: src/styles/sidebar.css [hunk 1]
```diff
@@ -1,29 +1,66 @@
...
+
+  .sidebar-pages {
+    display: none;
+  }
+
+  .sidebar-links {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
```
Comment (Adarsh-Patel-Dev) [inline src/styles/sidebar.css:63]: Suggestion: you can make a utility class `flex-column-center` in order to avoid writing same piece of code

**Review by Adarsh-Patel-Dev — APPROVED** [review]
I could not find any issue with your code. 
UI and whole code is well written. You've followed best practices in your code. I got to learn so many new things like react player in your code.

File: src/router/utils/HelperFunctions.jsx [hunk 2]
```diff
@@ -12,10 +12,7 @@ export const categories = async (videoDispatch) => {
       });
     }
   } catch (error) {
-    Toast({
-      type: "error",
-      message: "Oops! An error occurred.Try again later.",
-    });
+    toast.error("Oops! An error occurred.Try again later.");
     console.log(error.message);
```
Comment (dev-prabhat) [inline src/router/utils/HelperFunctions.jsx:16]: remove console.log here

File: src/router/utils/HelperFunctions.jsx [hunk 3]
```diff
@@ -36,13 +33,10 @@ export const addToWatchlist = async (video, videoDispatch) => {
...
-      Toast({ type: "success", message: "Added to Watch Later" });
+      toast.success("Added to Watchlist");
     }
   } catch (error) {
-    Toast({
-      type: "error",
-      message: "Oops! An error occurred.Try again later.",
-    });
+    toast.error("Oops! An error occurred.Try again later.");
     console.log(error.message);
```
Comment (dev-prabhat) [inline src/router/utils/HelperFunctions.jsx:40]: same console.log here also

File: src/router/utils/HelperFunctions.jsx [hunk 4]
```diff
@@ -178,13 +156,10 @@ export const clearHistory = async (videoDispatch) => {
...
-      Toast({ type: "success", message: "Cleared History" });
+      toast.success("Cleared History");
     }
   } catch (error) {
-    Toast({
-      type: "error",
-      message: "Oops! An error occurred.Try again later.",
-    });
+    toast.error("Oops! An error occurred.Try again later.");
     console.log(error.message);
```
Comment (dev-prabhat) [inline src/router/utils/HelperFunctions.jsx:163]: remove all the console.log

File: src/styles/header.css [hunk 5]
```diff
@@ -79,3 +84,84 @@
...
+    position: absolute;
+    right: 0;
+    top: 1.5rem;
+    width: 8rem;
+    background-color: var(--LIGHT-NAV-BG);
+    padding: 1rem;
+    border-radius: 4px;
+    color: var(--CTAS-LIGHT);
+    border: thin solid var(--DELETE-BTN);
+    font-size: 14px;
```
Comment (dev-prabhat) [inline src/styles/header.css:153]: you can use rem when it's value is greater than 10px

File: src/styles/video.css [hunk 6]
```diff
@@ -1,9 +1,25 @@
+:root {
+  --PLAYER-WIDTH: 1024px;
```
Comment (dev-prabhat) [inline src/styles/video.css:2]: you can provide rem or em or in percentage here

**Review by dev-prabhat — APPROVED** [review]
Suggested some change to be implemented in CSS code, Logic is all good UI is neat

File: src/router/utils/HelperFunctions.jsx (hunk 4 above)
Comment (SinchanaMS) [reply src/router/utils/HelperFunctions.jsx:163]: Thank you. These have been kept intentionally.

**Comment by SinchanaMS** [comment]
Thank you for the reviews, merging to dev.

**Pull request merged**